includes/options/class-amp-options-manager.php

```diff
@@ -311,6 +347,10 @@ public static function validate_options( $new_options ) {
 			}
 		}
 
+		if ( isset( $new_options[ Option::READER_THEME ] ) ) {
+			$options[ Option::READER_THEME ] = sanitize_key( $new_options[ Option::READER_THEME ] );
```

Comment:
Should this not rather check to see if `$new_options[ Option::READER_THEME ]` is actually among the list of reader themes?